mimic/rest/cloudfeeds.py

```diff
@@ -0,0 +1,176 @@
+# -*- test-case-name: mimic.test.test_loadbalancer -*-
+"""
+Defines the control plane API endpoints for the Cloudfeeds Plugin.
+"""
+import json
+from uuid import uuid4
+from six import text_type
+from zope.interface import implementer
+from twisted.web.server import Request
+from twisted.plugin import IPlugin
+from mimic.rest.mimicapp import MimicApp
+from mimic.imimic import IAPIMock
+from mimic.catalog import Entry
+from mimic.catalog import Endpoint
+
+from mimic.model.clb_objects import (
+    GlobalCLBCollections, BadKeysError, BadValueError
+)
+from random import randrange
+
+from mimic.util.helper import invalid_resource, json_dump
+from characteristic import attributes
+
+
+Request.defaultContentType = 'application/json'
+
+
+@implementer(IAPIMock, IPlugin)
+class CloudFeedsApi(object):
+    """
+    This class registers the cloud feeds API in the service catalog.
+    """
+    def __init__(self, regions=["ORD"]):
+        """
+        Configures the API with a list of regions.
+
+        :param list regions: If provided, a list of strings, each providing a
+            name for a region (e.g., "ORD").  If not specified, defaults to
+            ["ORD"].
+        """
+        self._regions = regions
+
+    def catalog_entries(self, tenant_id):
+        """
+        Returns a list of cloud feeds entries.  Note that these are not
+        cloud feeds product endpoints; this is one step removed from those.
+        You'll need to GET from one of these URLs to see the catalog of
+        product endpoints.
+        """
+        return [
+            Entry(tenant_id, "rax:feeds", "cloudFeeds",
+                  [
+                      Endpoint(tenant_id, region, text_type(uuid4()), prefix="noversion")
```

Comment:
Non-blocking - just as a note, leaving out the prefix entirely means that it won't be present at all in the URL returned by identity.